### PR TITLE
165189021-update sonarcloud ignore

### PR DESCRIPTION
**Pull request by c-troha**

### CODE REVIEW

File: SonarQube.Analysis.xml [hunk 1]
```diff
@@ -5,4 +5,6 @@
     <Property Name="sonar.projectVersion">0.3.0</Property>
     <Property Name="sonar.host.url">https://sonarcloud.io/</Property>
     <Property Name="sonar.organization">revaturexyz</Property>
+    <Property Name="sonar.projectKey">housing:forecast</Property>
```
Comment (escalonn) [inline SonarQube.Analysis.xml]: why is the project key inserted - this has to be provided on the command line (in dockerfile-ci) anyway, what good does it do here

File: src/Housing.Forecast.Testing/Context/SnapshotTests.cs [hunk 2]
```diff
@@ -18,7 +18,7 @@ public void DefaultSnapshotInvalidTest()
         }
 
         [Fact]
-        public void SnapshotValidTest()
+        public void SnapshotValidTest()
```
Comment (escalonn) [inline src/Housing.Forecast.Testing/Context/SnapshotTests.cs]: 🤔

Comment (c-troha) [reply src/Housing.Forecast.Testing/Context/SnapshotTests.cs]: this was Lee pushing a "blank" commit it seems

Comment (escalonn) [reply src/Housing.Forecast.Testing/Context/SnapshotTests.cs]: ok... let's get rid of it then

**Comment by escalonn** [comment]
make sure branch is up to date - it is trivial merge in this case so i just clicked the button
(this lets us make sure the exact same code will build before it arrives to broker, instead of a slightly different code)

**Pull request merged**